extras/ams_wrapper/tests/test_classification_type.py

```diff
@@ -16,43 +16,43 @@
 
 import pytest
 
-from src.api.types import Tag, SingleClassification, Classification
+from src.api.types import Tag, SingleClassification, Classification, Attribute
 
 def test_single_classification():
     expected_dict = {
-        "tag": {
+        "attributes": [{
+            "name": "type",
             "value": "car",
             "confidence": 0.97
-        },
+        }]
     }
-
-    tag = Tag("car", 0.97)
-    single_classification = SingleClassification(tag)
+    attribute = Attribute("type", "car", 0.97) 
+    attribs = []
+    attribs.append(attribute)
+    single_classification = SingleClassification(attribs)
     assert expected_dict == single_classification.as_dict()
 
 def test_entity():
```

Comment:
You could change name here to `test_classification`

Reply:
Done.